Nomster/CreateSuggestionViewController.swift

```diff
@@ -86,11 +86,11 @@ class CreateSuggestionViewController : UITableViewController {
         }
         
         // Dummy values
-        let image = "nomster-logo-large"
+        let image = ""
```

Comment:
You're right. I reverted my change and added a condition to allow local and remote images for now. So we can still also use fixtures.